yt/frontends/athena/data_structures.py

```diff
@@ -632,12 +632,7 @@ def _parse_parameter_file(self):
 
     @classmethod
     def _is_valid(cls, filename, *args, **kwargs):
-        try:
-            if "vtk" in filename:
-                return True
-        except Exception:
-            pass
-        return False
+        return "vtk" in os.path.basename(filename)
```

Comment:
I'm 99% sure that it's not even *in* the filename, but it's supposed to be `.vtk` extension. So either:

```
filename.endswith(".vtk")
```

or

```
os.path.splitext(filename)[-1] == ".vtk"
```

would be even more strict and appropriate.

Reply:
yeah I went your your first suggestion in my first iteration, then changed my mind in fear that it was not conservative enough, but I'm glad that you made me change my mind _again_, !

Reply:
forced pushed !